### ISSUE TITLE
Add round Omega as stylistic alt in Ponomar

### ISSUE THREAD

**Issue by aleslavista** [typiconman/fonts-cu#37]
Virtually all capital letters in Ponomar have serifs, however capital OMEGA U+460 conspicuously lack them, I think they should be placed on top of the three ascenders, as happens with capital letter SHA, for consistency reasons. Incidentally I wonder why capital letter OT U+47E features a different shape although the lower-case is clearly derived from lower-case OMEGA.

May I suggest you unify the two capital shapes? Admittedly I'd like U+460 to look like U+47E (minus the T on top, of course).

Regards,

**Comment by starover77** [comment]
Dear aleslavista, It is the tradition for Orthodox Christians to devote the entire first week of Great Lent (Velikii Post) to spiritual pursuits. Members of the Ponomar Project will save these messages and work on any issues after the Sunday of Orthodoxy, when we have time to address them. (The period from Lazarus Saturday until after Thomas Sunday is likewise a period when most of us take a break from our computers.) Thank you for your input and for your patience. We will get back to you as soon as we can. - Nikita

**Comment by typiconman** [comment]
The Ponomar font is designed to represent the usage in current liturgical books of the Russian Orthodox Church. What you propose is absolutely not the case in any liturgical book (see picture).

Your idea *could* be used in a stylzed font, such as Monomachus, which is not intended to faithfully reproduce any existing typeface, but rather is just intended to provide all Unicode characters and "look pretty".
![screenshot from 2018-02-21 21 33 48](https://user-images.githubusercontent.com/5428695/36498501-7f0508a0-174f-11e8-889b-1d83c16f94c6.png)

**Comment by aleslavista** [comment]
Hello,

Thanks for your answer. To be honest, I'd come to your same conclusion about the shape of capital letter OT after having checked my Church Slavonic Bible (see attachment). But I refrained from posting here since this morning I received a message about respecting the Orthodox Great Lent, so I complied. However, now that we've ascertained that liturgical books mandate such shape, I think you _shouldn't_ use my "idea" in Monomachus, I don't deem it appropriate to deviate from the standard glyphs we see in printed Church Slavonic books. 

However the shape of capital letter OT was just a remark I made out of curiosity for this seeming discrepancy: the core question concerned the **lack of serifs in capital OMEGA**.

**Comment by starover77** [comment]
Sorry, I didn't mean to say that nobody is checking their email this week, but I want to let you know that if you didn't get a response, it doesn't mean that we are purposefully ignoring you. I wish you a spiritually profitable Great Lent.

BTW: I concur with Aleksandr's reply about the shapes of the Capital Letter OT. I have seen the wrong shape used a few times, but only in "samizdat" (do-it-yourself-at-home) amateur desktop publishing. I have also seen this and other "theoretical" or "fabricated" (undocumentable) characters used in early digital fonts (c. 1985-2000).

**Comment by aleslavista** [comment]
No problem, starover77 and thank you for the wish.

As I said above, the shape of capital OT was just a side issue, I really wanted to discuss the lack of serifs in capital OMEGA.

About that, I just reaized Monomakh **does** have serifs on capital OMEGA, and this makes its look more "consistent", if you will, with other capital letters, particularly capital SHA, unlike Ponomar's capital OMEGA which lacks them. Of course I know Monomakh is a

> "stylzed font ... which is not intended to faithfully reproduce any existing typeface, but rather is just intended to provide all Unicode characters and "look pretty"

and that

> The Ponomar font is designed to represent the usage in current liturgical books of the Russian Orthodox Church. What you propose is absolutely not the case in any liturgical book (see picture).

but I'm just pointing out what seems to me a glaring inconsistency. For a high-quality font like Ponomar, it just makes me cringe to see capital SHA with serifs and capital OMEGA without, with all due respect for tradition, this really does give me the impression of a "samizdat" look. In my humble opinion, of course.
![monomakh ponomar](https://user-images.githubusercontent.com/33322424/36505294-54aefe72-1753-11e8-8411-d9a5c6800485.png)

**Comment by starover77** [comment]
OK, I just took some time to go through my extensive library of Church Slavonic books. I have over one hundred early editions before the late 1600s (poluustav typography) in digital format, and almost the same amount printed from the late 1600s to the present (Synodal and Kievan "post-poluustav" typography). I also have over 100 printed books in my home library, which range from 1637 to 1918.

My observations are:

**Poluustav:** In none of the poluustav editions was the Capital Letter Omega (U+0460) designed with serifs. In addition, both Muscovite and Kievan poluustav used the same shape... almost exactly as you see it in the Ponomar Unicode font. The only difference that I can see, is that the tops of the three curved lines are not supposed to be horizontally flat, but are traditionally beveled, and thus placing serifs on this character would look very strange. (See image below.)
![cyrillic capital letter omega - old style](https://user-images.githubusercontent.com/10853367/36512401-3f71b308-1738-11e8-8ea9-7eebad404bf8.gif)

**Synodal (Post-Poluustav):** In general, **Muscovite** publications continued to use the same character shape as above, except that it gradually became the practice to design the Capital Letter Omega with horizontally flat tops, instead of with the beveled terminus. The practice of flattening off the top of this character is now the modern default way to design it, but it is still very rare to see serifs added. The example that you see in the Monomakh font is an aberration to the tradition of Slavonic font design. (But of course, it was designed by a modern academic for use in journals, not by someone who was trying to replicate traditional typographer for use in church literature.) I suppose that you could say that it's a modern artistic interpretation of what someone thinks the character SHOULD look like.

On the other hand, in **Kievan** publications that were printed around 1700 to the present, the older shape of the Capital Letter Omega was changed to the rounded form:
![cyrillic capital letter omega - new style](https://user-images.githubusercontent.com/10853367/36512964-023992dc-173b-11e8-93b0-5dcec5e8709c.gif)
Gradually this new shape started being used in Muscovite publications, although it was never a complete conversion. It seems that it varied from one printing shop to another, and might have depended on where they bought their lead type or matrices from, or it might have been a matter of a preferred style by the typographer. (Of course, we will never know.)

So, in summary: The first image is the older, traditional form, while the second image is the modern-era Kievan form. Placing both forms in a single font is almost asking for trouble, as someone who doesn't know the difference will inevitably use the wrong form and create an anachronism. It is better to simply let the font style determine the character shape. If it's a font that reproduces Muscovite typography, the first shape should be placed at U+0460, and if it's a Kievan font, then the second shape should be placed at U+0460. I am against letting the user have a choice, unless we want to add the second form as a stylistic alternative. (But of course, that requires that the user should understand the difference, and I remain skeptical of that.)

**Comment by typiconman** [comment]
@aleslavista The captial Omega in books published by the Moscow Synodal Press (which is the typeface that Ponomar is supposed to reproduce) does not have serifs. (I think @starover77 says as much.)
However, since the Kievan editions use a rounded Omega, maybe we should place the rounded variant as a stylistic alternative (see here for example: http://old.stsl.ru/manuscripts/staropechatnye-knigi/0-5 ).
Of course, at some point, it would make sense to design a Kievan-looking font.

**Comment by aleslavista** [comment]
OK, thanks you so much to both of you for thoroughly delving into the topic. Yet, I still don't understand the _rationale_ for "denying" OMEGA serifs, given that there ought to be some criteria, typography isn't supposed to be random, right?. Oddly also Menaion's capital OMEGA seem to have serifs, even though it's supposed to faithfully reproduce USTAV manuscripts, before the invention of printing. Is it an aberration too, with no grounding on Slavonic font design?

![menaion](https://user-images.githubusercontent.com/33322424/36553355-65a27bba-17fc-11e8-8b82-b33c45a58e41.png)

Bearing in mind that serifs on top of capital OMEGA have no typographical history behind them, as far as Monomakh is concerned, I think you should either **remove** them, or, alternatively, maybe give OMEGA a "Kievian" look.

Again, thank you for satisfying my curiosity, I'm drawing up a dictionary of Codex Marianus' Gospel of John (it includes both Greek and CS correspondents of each word) so understandably I want my text to look carefully typeset.

Regards,

**Comment by typiconman** [comment]
@aleslavista I don't see a reason to remove serifs in Monomachus. It is a stylized font, and it has serifs.

Menaion is a different issue. The glyphs there are based on the Путятина минея and should be historically accurate. But the whole font, unfortunately, was poorly designed.

**Comment by aleslavista** [comment]
But you yourself said it's an aberration! I thought it was the logical thing to do since we've ascertained historical typography doesn't warrant it. Actually, giving it a "Kievian" shape would more justified than keeping the aberration since at least the latter does have a typographical history.

Concerning Menaion, since you deem it "poorly designed", that means you don't recommend my using it in my publication?

**Comment by typiconman** [comment]
@aleslavista Ponomar is intended to reproduce the Moscow typeface, where we do not have the rounded Omega. However, I see now reason why we cannot add the rounded Omega as a stylistic variant for anyone wishing to use a Kievan typeface. This is the only thing we can do now, since we don't have a separate Kievan font.

Concerning Menaion, unless you're trying to reproduce the exact look of the manuscript, you are probably better off using Monomachus. It's an "academic" font. It's what I use in my dissertation and papers.
But the Codex Marianus is in Glagolitic. Are you typesetting in Cyrillic transliteration?

**Comment by aleslavista** [comment]
I was talking about removing serifs from **Monomakh**, not Ponomar. As stylized as a font may be it still has to respect to follow some conventions and a rounded OMEGA certainly does have a history behind it, unlike a serified OMEGA.

Concerning Menaion, of course I'm working on Jagic's critical edition which is transliterated. Frankly speaking, I did plead with my publisher to allow me to use Glagolitic but he rebutted it'd make the work inaccessible to 99% of the readership. Thus, given that it's a transliteration, I'm not trying to reproduce the "exact look" of the manuscript, but I still want to have enough visual "gap" to bring out the fact that the Synodal Redaction comes almost a millennium after Codex Marianus. So I chose Menaion for Codex Marianus and Ponomar for the Synodal Redaction.

**Comment by starover77** [comment]
Whether the font is Monomachus, Monomakh, Putiata, PonomarUnicode, or some other font, we must first stop and consider a few things before we attempt to introduce changes to the design of a font. The first (and most important) consideration is whether or not we have a genuine right to alter the designer's work. In all cases, the Ponomar Project has been granted permission to RE-ENCODE the existing font to Unicode, add the necessary characters to make it Unicode-compliant, and add features that augment the original design. It has NEVER been our intent to redesign a font, just because we thought that the designer made some unfortunate choices in design.

Second, we have to consider the potential effects of making changes. I personally think it is in poor taste to alter someone else's artistic creation, or to overly-criticize their work. I have had people do this to me, taking my font designs and redesign them significantly, without my permission, and then adding their own name to the font and deleting my name, and in all honesty it felt like someone took a knife to me. It is legally called "intellectual property theft". There is a fine line between "tweaking" a font and extensively redesigning it, and I would rather err on the side of RESPECT to the font designer than risk gaining disrespect from my professional colleagues.

Having said all that, I am also not in favor of introducing changes to these fonts because I assume that they have already been tested by the original designers for consistency in appearance, evenness of weight, serifs where needed, and other factors. If a particular character is lacking serifs, then I assume that the designer tested it both with and without serifs, and made an _artistic decision_ to omit them because it looked more harmonious in context. I choose to respect the designer's decision.

In all honesty, after looking at all the fonts in question, there are other design issues that bother me far more than whether the capital Omega has serifs or not. But whether I like the fonts or not, I acknowledge that they already serve a valuable purpose just as they are, and they will get the job done with integrity.

**Comment by typiconman** [comment]
@starover77 I don't understand what the problem is. The license (SIL OFL) allows us to do anything with the font. We've already redesigned Ponomar quite considerably and some characters differ significantly from the original Hirmos UCS font.

**Comment by starover77** [comment]
@typiconman  I understand what you are saying about having the right to make such a change (adding serifs to the capital letter Omega), but since there is no established practice of this that can be observed in authentic typographical practice from the beginning of printing until the present (not considering fonts that were designed during the last 30 years for use in desktop publishing), I don't think we should introduce an unwarranted innovation. (And speaking personally, I think it has a certain graceful and harmonious appearance as it currently stands. Adding serifs would make it look "unbalanced"... too heavy at the top and too light at the bottom.)

Also, if you DO try to add serifs to the character, you would need to redesign the character so that the middle and right serifs would not be touching, since they come very close together. This would look confusing, particularly if used in actual printing (since you have to consider that ink has a small percentage of dispersion when it comes into contact with paper in offset printing).

**Comment by aleslavista** [comment]
Hello,

I think I've not been able to bring the point home that a serified capital OMEGA is **no longer an issue** since we've ascertained it has no place in proper CS typography. My proposal shifted to giving capital OMEGA in Monomakh a rounded shape. That's why I opened a new post to avoid confusion.

Also please remember I'm just posting my comments here, I never tried to foist upon you my views or forced you to redesign the font let alone disrespecting your accomplishments.

**Comment by starover77** [comment]
Sorry, there is a lot going on right this week. I wasn't aware that the subject was closed, as the thread remained opened for further discussion. Can we close this subject now?

**Comment by typiconman** [comment]
I added the round form of U+0460 as a stylistic alternative (accessed via salt). The form is encoded in the PUA at U+E2EA.
![Screenshot from 2019-03-17 14-23-34](https://user-images.githubusercontent.com/5428695/54489725-43cdeb00-48c0-11e9-98d8-e6fe3cecfb97.png)

**Comment by typiconman** [comment]
Fixed.
